examples/proc_FID_nutation.py

```diff
@@ -47,21 +47,15 @@
     )
     s.fit()
     fit = s.eval(500)
-    # {{{ weirdness with units of subplots means fitting is in s*sqrt(W) and
-    #     for plotting I convert to us*sqrt(W)
-    s["beta"] /= 1e-6
-    fit["beta"] /= 1e-6
-    # }}}
-    fl.plot(s, "o", human_units=False, ax=ax3)
-    fl.plot(fit, human_units=False, ax=ax3)
-    ax3.set_title("Integrated and fit")
-    ax3.set_xlabel(r"$\beta$ / $\mathrm{\mu s \sqrt{W}}$")
-    beta_90 = s.output("beta_ninety") / 1e-6
-    ax3.axvline(beta_90)
-    ax3.text(
+    fl.plot(s, "o", ax=ax_last)
+    fl.plot(fit, ax=ax_last)
+    ax_last.set_title("Integrated and fit")
+    beta_90 = s.output("beta_ninety")
+    ax_last.axvline(beta_90)
+    ax_last.text(
```

Comment:
this was a problem that needed to be fixed in pyspecdata.  The figure list was really set up for a one dataset per figure model, which we're obviously messing with here.